content/posts/a_confusing_build_failure.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "A confusing build failure"
+date: 2019-11-07T01:00:09+02:00
+draft: true
+tags: ["testing", "continuous integration"]
+---
+
+{{< figure src="/images/posts/broken_builds/judge-dredd-under-arrest.jpg" title="" >}}
+
+Nowadays it has become quite a common practice to build and check software on machines other than local workstations, most of the time called CI servers(CI stands for [Continuous Integration](https://martinfowler.com/articles/continuousIntegration.html)). This can be done manually or automatically, usually triggered by a pull/merge request or a direct push on the main trunk(in git terminology most of the times named *master*).
```

Comment:
```suggestion
Nowadays, it has become quite a common practice to build and check software on machines other than local workstations, most of the time called CI servers(CI stands for [Continuous Integration](https://martinfowler.com/articles/continuousIntegration.html)). This can be done either manually or automatically, usually triggered by a pull/merge request or a direct push on the main trunk(in git terminology most of the times named *master*).
```